FrameworkSystem/Client/SystemAdministratorClientCLI.py

```diff
@@ -9,6 +9,8 @@
 import sys
```

Comment:
Is it possible that this functionality is extracted from here and above, and put somewhere common?